### DISCUSSION TITLE
Snakemake is needed for the project？How do you use Snakemake?

### DISCUSSION THREAD

**Discussion by mome1024** [bitextor/bitextor#220] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/51071095/132662601-24bb4eab-e930-4b2f-a7bf-7575b70aa74b.png)

Snakemake is needed for the project？How do you use Snakemake?

**Comment by zuny26** [comment, marked as answer]
Hello @mome1024

Snakemake is a workflow management tool that Bitextor executes. For the basic usage of Bitextor you don't need to know any details about it.

The `Snakefile` in the help message refers to the configuration file (.yaml) that Bitextor needs to define some parameters/variables of the workflow. You can find more information about this file in the README of your version of Bitextor.

**Reply by mome1024** [reply]
What configuration should I add to get Bitextor running?

**Reply by cgr71ii** [reply]
Hi, @mome1024 ! I know I'm a little bit late, but just in case it is useful for you or for someone else, here is a little explanation:

Bitextor has many configuration options, and you can configure them using the CLI through `--config` or provide a YAML file with the directives which you would provide to the `--config` flag. In order to provide a YAML file, you can use `-C /path/to/file`. The explanation about all the configuration directives may be found [here](https://github.com/bitextor/bitextor/blob/master/docs/CONFIG.md). Also, you have a minimal YAML configuration file [here](https://github.com/bitextor/bitextor/blob/master/docs/config/basic.yaml), which might help you to start and make Bitextor work (if I'm not wrong you should just need to change the paths to paths from your system).

I hope it helps!
